packages/forms-web-app/src/dynamic-forms/s78-appeal-form/journey.js

```diff
@@ -170,7 +170,7 @@ const sections = [
 		.addQuestion(questions.inspectorAccess)
 		.addQuestion(questions.healthAndSafety)
 		.addQuestion(questions.enterApplicationReference)
-		.addQuestion(questions.planningApplicationDate())
+		.addQuestion(questions.planningApplicationDate)
```

Comment:
These used to be functions with the intention of dynamically generating a date 60 days in the future or past. When I tested this it actually wasn't working as intended and showed 60 days before/after when the container started, as opposed to when the page loaded. Happy to loop back and have another go at making that feature work as intended but for now it just seemed easier to make all the question signatures the same